x-pack/plugins/infra/server/lib/alerting/metric_threshold/lib/evaluate_alert.ts

```diff
@@ -239,7 +239,7 @@ const getMetric: (
           : 0
       ),
     };
-  } catch (e) {
+  } catch (e: any) {
```

Comment:
Would `unknown` work the same way here for our purposes? Might be safer/save us an additional `any` toward the overall count.

Reply:
Sadly no, it looks like a Typescript upgrade or something introduced a new error which is precisely complaining about the fact that `e` is `unknown`.

Reply:
Fun! Thanks